es-core/src/Window.cpp

```diff
@@ -335,8 +335,8 @@ void Window::setHelpPrompts(const std::vector<HelpPrompt>& prompts, const HelpSt
 				// yes, it has!
 
 				// can we combine? (dpad only)
-				if((it->first == "up/down" && addPrompts.at(mappedTo->second).first == "left/right") ||
-					(it->first == "left/right" && addPrompts.at(mappedTo->second).first == "up/down"))
+				if((strcmp(it->first, "up/down") == 0 && addPrompts.at(mappedTo->second).first == "left/right") ||
```

Comment:
If I'm not mistaken, all quantities are `(const) char *` here, in which case `it->first == "up/down"` doesn't do what you'd expect: It really compares the addresses of the `char *`. `strcmp` actually performs string comparison.